spec/graphql/analysis/max_query_depth_spec.rb

```diff
@@ -48,7 +56,8 @@
 
   describe "when the max depth isn't set" do
     before do
-      schema.max_depth = nil
+      # Yuck - Can't override GraphQL::Schema.max_depth to return nil if it has already been set
+      schema.define_singleton_method(:max_depth) { |*| nil }
```

Comment:
Oops! If we need this for something other than test setup one day, we can do something like 

```
def max_depth(new_max_depth = :unset) 
  if new_max_depth == :unset # Nothing was given 
    # read `@max_depth` with inheritance 
  else 
    @max_depth = new_max_depth 
  end 
end 
```